cypress/integration/plp.test.js

```diff
@@ -63,11 +80,15 @@ describe('Search page Filters and Sorting options', () => {
   })
 
   it('Sort products by price_desc', () => {
+    cy.visit(pages.collection, options)
+    cy.waitForHydration()
+    const priceId = '[data-testid="store-card"] [data-testid="price"]'
+
     cy.getById('search-sort')
       .should('exist')
       .select('price_desc')
       .then(() => {
-        cy.getById('price').should(($prices) => {
+        cy.get(priceId).should(($prices) => {
```

Comment:
same here?

Reply:
Yep, same kind of the test so see my [previous answer](https://github.com/vtex-sites/base.store/pull/131#discussion_r792877686).